src/content/en/fundamentals/security/credential-management/save-forms.md

```diff
@@ -140,10 +137,14 @@ or proceed to the personalized page.
       e.preventDefault();
 
       // Try sign-in with AJAX
-      fetch(/'signin', {
+      let form = new FormData();
+      form.append('email', c.id);
+      form.append('password', c.password);
+
+      return fetch('/signin', {
         method: 'POST',
-        body: new FormData(e.target),
-        credentials: 'include'
+        credentials: 'include',
+        body: form
       }).then(res => {
```

Comment:
I rolled the changes back-- but good for you to have one last look over code to make sure it's OK.